recipe/meta.yaml

```diff
@@ -185,6 +185,9 @@ outputs:
         - nccl ~=2.16           # [linux]
         - cutensor ~=2.0
       run:
+        {% if cuda_major >= 12 %}
+        - cuda-cudart-dev
+        {% endif %}
```

Comment:
Think CuPy might be looking in `$PREFIX/include`. So we would want this

However ideally it would be able to work with the `targets` structure and we could then use `cuda-cudart-dev_{{ target_platform }}` here

Probably there is something else needed in how CuPy handles search for headers

Reply:
Addressed in PR: https://github.com/cupy/cupy/pull/8520